lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimation.kt

```diff
@@ -118,17 +113,19 @@ fun LottieAnimation(
     }
 
     if (composition == null || composition.duration == 0f) return
-    SideEffect {}
 
     Canvas(
         modifier = Modifier
             .maintainAspectRatio(composition)
             .then(modifier)
     ) {
         drawIntoCanvas { canvas ->
-            drawable.progress = state.progress
-            drawable.setOutlineMasksAndMattes(state.outlineMasksAndMattes)
-            drawable.isApplyingOpacityToLayersEnabled = state.applyOpacityToLayers
+            drawable.apply {
```

Comment:
nit: I don't think the drawable.apply makes things easier to read here. It just adds 2 lines of code an some cognitive overhead to figure out what the `this` is.

Reply:
Hmm I just compared and think it's easier to read that way - compare it to

```kotlin
drawable.progress = state.progress
drawable.setOutlineMasksAndMattes(state.outlineMasksAndMattes)
drawable.isApplyingOpacityToLayersEnabled = state.applyOpacityToLayers
drawable.enableMergePathsForKitKatAndAbove(state.enableMergePaths)
```

I'm expecting some other things will be added here too over time, so it makes sense to scope it to the `apply` IMO. Unless you have strong concerns over this I'll keep it :)

Reply:
@jossiwolf although there is no "right" here, I do prefer avoiding `apply` unless it is a very large block. The `drawable` prefix is roughly analogous to the extra indentation from the apply block but it makes it immediately clear what `this` is. Without it, you could silently mix and match calls this this@LottieDrawable and this@LottieAnimation which could lead to ambiguous situations.

Reply:
Yep, let's go with what's more agnostic to the codebase then! I'll push a commit tomorrow:)